lib/modules/dosomething/dosomething_reportback/dosomething_reportback.admin.inc

```diff
@@ -22,11 +22,11 @@ function dosomething_reportback_admin_config_form($form, &$form_state) {
     '#default_value' => variable_get('dosomething_reportback_log', FALSE),
     '#description' => t("Logs Reportback activity. This should be disabled on production."),
   );
-  $form['dosomething_reportback_display_permalink'] = array(
+  $form['dosomething_reportback_display_permalink_share'] = array(
     '#type' => 'checkbox',
-    '#title' => t('Enable permalink page.'),
-    '#default_value' => variable_get('dosomething_reportback_display_permalink', FALSE),
```

Comment:
Its OCD, but can we delete any variables we are no longer using in an update N hook?